scripts/common

```diff
@@ -10,6 +10,9 @@ set -o pipefail
 # https://docs.rs/env_logger
 export RUST_LOG="${RUST_LOG:-info}"
 
+# Reduce build nondeterminism by remapping paths so that they don't include local segments.
+export RUSTFLAGS="--remap-path-prefix=$PWD=/ ${RUSTFLAGS:-}"
```

Comment:
Shouldn't this flag be `remap-path-prefix-from=` with `remap-path-prefix-to=` ?
https://doc.rust-lang.org/1.24.0/unstable-book/compiler-flags/remap-path-prefix.html

Reply:
NVM, it's the same flag